guide/src/migration.md

```diff
@@ -147,8 +147,49 @@ Notable features of this new trait:
 - `()` (unit) is now only special handled in return position and otherwise converts into an empty `PyTuple`
 
 All PyO3 provided types as well as `#[pyclass]`es already implement `IntoPyObject`. Other types will
-need to adapt an implementation of `IntoPyObject` to stay compatible with the Python APIs.
+need to adapt an implementation of `IntoPyObject` to stay compatible with the Python APIs. In many cases
+the new [`#[derive(IntoPyObject)]`](#intopyobject-derive-macro) macro can be used instead of
+[manual implementations](#intopyobject-manual-implementation).
 
+#### `IntoPyObject` derive macro
```

Comment:
I think some of this detail would be better suited in the normal guide (maybe near where the `FromPyObject` derive is documented). Possibly here we can show an example `IntoPy` implementation and then show a derived `IntoPyObject`, and leave the detail for the normal guide?

Reply:
I gave this a go